spec/integration/api_request_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
+
+describe "API requests for content items", :type => :request do
```

Comment:
I'm not quite clear on the scope of this test.  Aren't all the GET requests API requests for content items?